### PR TITLE
Recipes page list

### PR DESCRIPTION
**Pull request by dosterius**
Recipes page + navigation

### CODE REVIEW

File: app/views/recipes/index.erb [hunk 1]
```diff
@@ -0,0 +1,58 @@
+ <div class="container">
+            <div class="blog-post">
```
Comment (ozimeu) [inline app/views/recipes/index.erb]: bad indentation here

File: app/views/recipes/index.erb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+                <h1>Recipes</h1>
+
+                <div class="tab">
+                        <a href="http://localhost:3000/recipes/new"><button class="tablinks" >Compose your own recipe</button><a>
```
Comment (ozimeu) [inline app/views/recipes/index.erb]: you shouldn't use hardcoded paths like that, please use the methods from routes

File: app/views/recipes/index.erb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                        <a href="http://localhost:3000/users/sign_up"><button class="tablinks" >Sign Up</button><a>
+                        <button class="tablinks" onclick="openCity(event, 'Paris')">Search for recipe</button>
+                </div>
+
+
+                    <!-- <div class="menu-links">
+                    <a href="http://www.onet.com" target="_blank">Compose your own recipe</a>
+                    
+                    <a href="logo.jpg" target="_blank">Search for recipe   </a>
+
```
Comment (ozimeu) [inline app/views/recipes/index.erb]: do we need these comments here?

File: app/views/recipes/index.erb [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                    </p>
+                    <%= image_tag("beer.jpeg", :alt => "beer photo") %>
```
Comment (ozimeu) [inline app/views/recipes/index.erb]: 👍 using the tag helper :)

File: app/views/recipes/index.erb (hunk 1 above)
Comment (dosterius) [reply app/views/recipes/index.erb]: fixed

File: app/views/recipes/index.erb (hunk 2 above)
Comment (dosterius) [reply app/views/recipes/index.erb]: I've deleted it for a while ... we will add links and navigation later on

File: app/views/recipes/index.erb (hunk 3 above)
Comment (dosterius) [reply app/views/recipes/index.erb]: no.. deleted:)

File: app/views/recipes/index.erb (hunk 4 above)
Comment (dosterius) [reply app/views/recipes/index.erb]: :)

**Review by ozimeu — APPROVED** [review]
:+1:

**Pull request merged**